src/protocols/ssh/ssh.h

```diff
@@ -89,6 +89,12 @@ typedef struct guac_ssh_client {
      * Lock dictating access to the SSH terminal channel.
      */
     pthread_mutex_t term_channel_lock;
+    
+    /**
+     * Condition used when SSH client thread needs to wait for Guacamole
+     * client response.
```

Comment:
Same here - I think the usage of this condition should much more specific than an arbitrary client response of some unknown kind. It would be better to be explicit in the naming and documentation of this and its corresponding flags.

Reply:
Okay, I've switched this to be more specific.